### ISSUE TITLE
update event is called when the setEditable is modified 

### ISSUE THREAD

**Issue by GitHubish** [ueberdosis/tiptap#3594]
### What’s the bug you are facing?

Since I switched to version 209, I used to use the packages in their "own version" so before the harmonization of versioning. I have a bug, I have a system allowing to save the text entered during the destruction of my component in case the user would not have pressed voluntarily on the save button. But here, I see that the update event is called when I do a setEditable. And no argument allowing to set the emitUpdate to false on this method contrary to the setContent.

Why this change? How to solve my problem?

### Which browser was this experienced in? Are any special extensions installed?

Chrome, Firefox, Safari, Brave

### How can we reproduce the bug on our side?

```
editor.setEditable(true); // ON INIT IS DISABLED
editor.chain().setContent("MY_CONTENT", false).run();

//[...]
editor.on("update", updateOutput);  // ==> method called
```

### What did you expect to happen?

An argument should be available not to emit an update event when calling the setEditable or the event should not be called if it has no interest - specifically the content has not changed 

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by Rhys-T** [comment]
Possible duplicate of #3289. Might be fixed by #3301.

**Comment by GitHubish** [comment]
Problem solved with version beta 210. Thanks tiptap team ;)
